plugin/inlay_hint.py

```diff
@@ -162,10 +170,20 @@ def format_inlay_hint_label(inlay_hint: InlayHint, session: Session, phantom_uui
                 }
             })
             value += f'<a href="{inlay_hint_click_command}">'
-        value += html.escape(label_part['value'])
+        raw_label = label_part['value']
+        truncated = len(raw_label) > remaining_truncate_limit
+        if truncated:
+            truncated_label = raw_label[:remaining_truncate_limit] + '…'
+        else:
+            truncated_label = raw_label
+
```

Comment:
more python-like

```suggestion
        truncated_label = raw_label[:remaining_truncate_limit] + '…' if truncated else raw_label
```

Also we have convention in the code to not add blank links in function bodies (with some exceptions and some places where someone forgot to follow it ;)).

Reply:
Ternary operator: [9f0f3c7](https://github.com/sublimelsp/LSP/pull/2514/commits/9f0f3c712b4be52fe5f5d38a8310f223e3e6e871)
Removal of blank lines: [1a5bae4](https://github.com/sublimelsp/LSP/pull/2514/commits/1a5bae46dcac82e7cc0336f091407c5da3d528f1)